packages/ckeditor5-html-support/CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+Contributing
+========================================
```

Comment:
AFAICS we repeat the same mistake in all packages. The number of `=` characters should be equal to the "Contributing" word length.

Let's ignore this fact. The change must be applied to all packages.